src/Widgets/NavigationWidget/NavigationWidgetEditingConfig.ts

```diff
@@ -3,9 +3,20 @@ import { NavigationWidget } from './NavigationWidgetClass'
 import Thumbnail from './thumbnail.svg'
 
 provideEditingConfig(NavigationWidget, {
-  attributes: { logOutLabel: { title: 'Log-out menu item label' } },
+  attributes: {
+    logOutLabel: { title: 'Log-out menu item label' },
+    slimDesign: {
+      title: 'Slim design?',
+      description: 'Shows only the meta navigation, not the "main" navigation.',
```

Comment:
* [ ] Why is main in quotes? Maybe use _hierarchy_ to make clearer what main refers to?